.github/workflows/buf-ci.yaml

```diff
@@ -0,0 +1,18 @@
+name: buf
+on:
+  push:
+    branches:
+      - main
```

Comment:
Trigger push only on main, same as current.

Reply:
I don't think we have a good reason to deviate from what we recommend to others.

Reply:
Enabled sync: push and delete for branches.